Traceback (most recent call last):
  File "/home/hoan/Desktop/Malte/MalteVSC/dodo_Genesis_RL/dodo_train.py", line 305, in <module>
    main()
  File "/home/hoan/Desktop/Malte/MalteVSC/dodo_Genesis_RL/dodo_train.py", line 291, in main
    runner = CustomRunner(env, train_cfg, log_dir, device=gs.device)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hoan/Desktop/Malte/MalteVSC/dodo_Genesis_RL/dodo_train.py", line 223, in __init__
    super().__init__(env, train_cfg, log_dir, device)
  File "/home/hoan/miniconda3/envs/dodolab/lib/python3.12/site-packages/rsl_rl/runners/on_policy_runner.py", line 32, in __init__
    obs, extras = self.env.get_observations()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DodoEnv' object has no attribute 'get_observations'
Traceback (most recent call last):
  File "/home/hoan/Desktop/Malte/MalteVSC/dodo_Genesis_RL/dodo_train.py", line 305, in <module>
    main()
  File "/home/hoan/Desktop/Malte/MalteVSC/dodo_Genesis_RL/dodo_train.py", line 291, in main
    runner = CustomRunner(env, train_cfg, log_dir, device=gs.device)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hoan/Desktop/Malte/MalteVSC/dodo_Genesis_RL/dodo_train.py", line 223, in __init__
    super().__init__(env, train_cfg, log_dir, device)
  File "/home/hoan/miniconda3/envs/dodolab/lib/python3.12/site-packages/rsl_rl/runners/on_policy_runner.py", line 32, in __init__
    obs, extras = self.env.get_observations()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DodoEnv' object has no attribute 'get_observations'

[38;5;9m[Genesis] [13:04:22] [ERROR] AttributeError: 'DodoEnv' object has no attribute 'get_observations'[0m
